src/pollyCLI/pollyintegration.cpp

```diff
@@ -2,6 +2,10 @@
 #include <common/downloadmanager.h>
 #include <common/logger.h>
 #include <QTemporaryFile>
+#include "core/libmaven/mzUtils.h"
+#include "json.hpp"
+
+using json = nlohmann::json;
```

Comment:
We could have used Qt's own JSON parsing library. But this is not that big of a deal.